app/src/main/java/app/grapheneos/pdfviewer/fragment/PasswordPromptFragment.kt

```diff
@@ -0,0 +1,78 @@
+package app.grapheneos.pdfviewer.fragment
+
+import android.app.Dialog
+import android.content.DialogInterface
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextUtils
+import android.text.TextWatcher
+import android.view.LayoutInflater
+import android.view.WindowManager
+import android.widget.Button
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.DialogFragment
+import app.grapheneos.pdfviewer.PdfViewer
+import app.grapheneos.pdfviewer.R
+import app.grapheneos.pdfviewer.databinding.PasswordDialogFragmentBinding
+import com.google.android.material.textfield.TextInputEditText
+
+class PasswordPromptFragment : DialogFragment() {
+
+    private lateinit var passwordDialogFragmentBinding: PasswordDialogFragmentBinding
```

Comment:
`passwordDialogFragmentBinding` property is used only in `onCreateDialog`, remove it

Reply:
done.